plugin/teksi_wastewater/interlis/gui/interlis_export_settings_dialog.py

```diff
@@ -106,3 +112,10 @@ def selected_model(self):
 
     def selected_models(self):
         return self.export_model_selection_comboBox.currentData()
+
+    @property
+    def selected_orientation(self):
```

Comment:
```suggestion
    def labels_orientation_offset(self):
```
I would rename it label orientation offset or similar for more readability